c2corg_ui/static/js/documentediting.js

```diff
@@ -265,10 +267,9 @@ app.DocumentEditingController.prototype.submitForm = function(isValid) {
  */
 app.DocumentEditingController.prototype.successSave_ = function(response) {
   // redirects to the document view page
-  var url = '/{module}/{id}/{culture}'
-      .replace('{module}', this.module_)
-      .replace('{id}', response['data']['document_id'])
-      .replace('{culture}', this.culture_);
+  goog.asserts.assert(this.culture_ !== null);
+  var url = app.utils.buildDocumentUrl(
```

Comment:
On the other hand the document URL is the main one (view page). For instance https://www.camptocamp.org/waypoints/12345/fr (by the way we will have to add the slug at the end some day).

Other URL are annex URLs: editing, history, archived versions, etc.